opentelemetry-sdk/src/metrics/internal/aggregator.rs

```diff
@@ -1,6 +1,19 @@
+use crate::{attributes::AttributeSet, metrics::data::Aggregation};
+use once_cell::sync::OnceCell;
+use opentelemetry_api::KeyValue;
 use std::sync::Arc;
 
-use crate::{attributes::AttributeSet, metrics::data::Aggregation};
+const STREAM_CARDINALITY_LIMIT: u32 = 2000;
+static STREAM_OVERFLOW_ATTRIBUTE_SET: OnceCell<AttributeSet> = OnceCell::new();
```

Comment:
Nit: could use lazy here to reduce boilerplate https://docs.rs/once_cell/latest/once_cell/sync/struct.Lazy.html